process/track/export_test.go

```diff
@@ -1 +1,333 @@
+
+func (bb *blockBalancer) SetNumPendingMiniBlocks(shardID uint32, numPendingMiniBlocks uint32) {
+	bb.setNumPendingMiniBlocks(shardID, numPendingMiniBlocks)
+}
+
+func (bb *blockBalancer) GetNumPendingMiniBlocks(shardID uint32) uint32 {
+	return bb.getNumPendingMiniBlocks(shardID)
+}
+
+// blockNotifier
+
+func (bn *blockNotifier) RegisterHandler(handler func(shardID uint32, headers []data.HeaderHandler, headersHashes [][]byte)) {
+	bn.registerHandler(handler)
+}
+
+func (bn *blockNotifier) CallHandlers(shardID uint32, headers []data.HeaderHandler, headersHashes [][]byte) {
+	bn.callHandlers(shardID, headers, headersHashes)
+}
+
+func (bn *blockNotifier) GetNotarizedHeadersHandlers() []func(shardID uint32, headers []data.HeaderHandler, headersHashes [][]byte) {
+	bn.mutNotarizedHeadersHandlers.RLock()
+	notarizedHeadersHandlers := bn.notarizedHeadersHandlers
+	bn.mutNotarizedHeadersHandlers.RUnlock()
+
+	return notarizedHeadersHandlers
+}
+
+// blockNotarizer
+
+func (bn *blockNotarizer) AddNotarizedHeader(shardID uint32, notarizedHeader data.HeaderHandler, notarizedHeaderHash []byte) {
+	bn.addNotarizedHeader(shardID, notarizedHeader, notarizedHeaderHash)
+}
+
+func (bn *blockNotarizer) GetNotarizedHeaders() map[uint32][]*HeaderInfo {
+	bn.mutNotarizedHeaders.RLock()
+	notarizedHeaders := bn.notarizedHeaders
+	bn.mutNotarizedHeaders.RUnlock()
+
+	return notarizedHeaders
+}
+
+func (bn *blockNotarizer) GetNotarizedHeader(shardID uint32, index int) data.HeaderHandler {
+	bn.mutNotarizedHeaders.RLock()
+	notarizedHeader := bn.notarizedHeaders[shardID][index].Header
+	bn.mutNotarizedHeaders.RUnlock()
+
+	return notarizedHeader
+}
+
+func (bn *blockNotarizer) CleanupNotarizedHeadersBehindNonce(shardID uint32, nonce uint64) {
+	bn.cleanupNotarizedHeadersBehindNonce(shardID, nonce)
+}
+
+func (bn *blockNotarizer) DisplayNotarizedHeaders(shardID uint32, message string) {
+	bn.displayNotarizedHeaders(shardID, message)
+}
+
+func (bn *blockNotarizer) GetLastNotarizedHeader(shardID uint32) (data.HeaderHandler, []byte, error) {
+	return bn.getLastNotarizedHeader(shardID)
+}
+
+func (bn *blockNotarizer) GetLastNotarizedHeaderNonce(shardID uint32) uint64 {
+	return bn.getLastNotarizedHeaderNonce(shardID)
+}
+
+func (bn *blockNotarizer) LastNotarizedHeaderInfo(shardID uint32) *HeaderInfo {
+	return bn.lastNotarizedHeaderInfo(shardID)
+}
+
+func (bn *blockNotarizer) GetNotarizedHeaderWithOffset(shardID uint32, offset uint64) (data.HeaderHandler, []byte, error) {
+	return bn.getNotarizedHeader(shardID, offset)
+}
+
+func (bn *blockNotarizer) InitNotarizedHeaders(startHeaders map[uint32]data.HeaderHandler) error {
+	return bn.initNotarizedHeaders(startHeaders)
+}
+
+func (bn *blockNotarizer) RemoveLastNotarizedHeader() {
+	bn.removeLastNotarizedHeader()
+}
+
+func (bn *blockNotarizer) RestoreNotarizedHeadersToGenesis() {
+	bn.restoreNotarizedHeadersToGenesis()
+}
+
+// blockProcessor
+
+// BlockTrackerHandlerMock
+
+type BlockTrackerHandlerMock struct {
```

Comment:
Mocks should have been inside process.mock subpackage. In order to achieve this, as stated in the previous BT review, the interfaces should have been unexported but the methods belonging to the interface should have been exported. Can remain like this for the time being, but in order to comply with th style used in the rest of the project, we should change those interfaces and move these mocks.

Reply:
Done

Reply:
Done